components/brave_extension/extension/brave_extension/background/events/cosmeticFilterEvents.ts

```diff
@@ -75,7 +75,11 @@ chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
       if (url === undefined) {
         break
       }
-      shieldsPanelActions.contentScriptsLoaded(tabId, url)
+      const frameId = sender.frameId
+      if (frameId === undefined) {
+        break
+      }
+      shieldsPanelActions.contentScriptsLoaded(tabId, frameId, msg.location.href)
```

Comment:
since we're now sending `msg.location.href` and not `tab.url`, should we change the check for `undefined` to check `msg.location.href` and ignore `tab.url` completely? Or is that `tab.url` check to see if we're in a valid loaded tab?